imports/api/users/users.js

```diff
@@ -181,6 +182,14 @@ Meteor.users.helpers({
   toString() {
     return this.displayOfficialName();
   },
+  personNameMismatch(communityId = getActiveCommunityId()) {
+    const membership = Memberships.findOne({ communityId, approved: true, active: true, personId: this._id, 'person.idCard.name': { $exists: true } });
+    const personName = membership ? membership.person.idCard.name : undefined;
+    if (!personName) return;
+    const nameMatch = compareNames({firstName: this.profile.firstName, lastName: this.profile.lastName}, {name: personName});
```

Comment:
Ezt en ugy irnam picit meg elegansabban, hogy compareNames(this.profile, membership.person.idCard) -- felteve persze hogy a compare fuggvenyed nem ir bele a parametereibe, de gondolom nem, mert a compare szo azt suggalja, hogy csak olvas belole, nezegeti, es nezgetes alapjan van egy visszateresi erteke.